hazelcast/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -1150,6 +1153,32 @@ public JobConfig setStoreMetricsAfterJobCompletion(boolean storeMetricsAfterJobC
         return this;
     }
 
+    /**
+     * Returns the maximum number of records that can be accumulated by any single
+     * {@link Processor} instance in the context of the job.
+     *
+     * @since 5.0
+     */
+    public long getMaxProcessorAccumulatedRecords() {
+        return maxProcessorAccumulatedRecords;
+    }
+
+    /**
+     * Sets the maximum number of records that can be accumulated by any single
+     * {@link Processor} instance in the context of the job.
+     * <p>
+     * For more info see {@link InstanceConfig#setMaxProcessorAccumulatedRecords(long)}.
+     * <p>
+     * If set, it has precedence over {@link InstanceConfig}'s one.
```

Comment:
We should document that `-1` means "not set, use member's value". And that it's the default.